tools/linux/uio/uio.mk

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2024, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# This Makefile snippet builds block UIO driver
+#
```

Comment:
Include the name of the thing(s) it builds, as somewhere they need to be added to a dependency in the top-level makefiles.

Reply:
This snippet actually builds libuio.a --- a library that will be linked against the actual drivers.

Reply:
Done https://github.com/au-ts/libvmm/pull/79/commits/d648c3778208ea2e4e3f443439db11041b7cf325